tox.ini

```diff
@@ -58,14 +58,14 @@ deps =
     coverage[toml]
     -r{toxinidir}/requirements.txt
 commands =
-    coverage run --source={[vars]all_path} \
+    coverage run --source={[vars]src_path} \
         -m pytest --ignore={[vars]tst_path}integration -v --tb native -s {posargs}
     coverage report
 
 [testenv:integration]
 description = Run integration tests
 deps =
-    pytest
+    pytest-operator
     juju
     pytest-operator.git
     -r{toxinidir}/requirements.txt
```

Comment:
I think if you were to run `tox -e fmt; tox -e lint` you'd find a few issues in docstrings etc :)